app/src/main/res/layout/fragment_home.xml

```diff
@@ -35,4 +35,13 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:srcCompat="@drawable/ic_add_foreground" />
 
+    <TextView
+        android:id="@+id/LabelforReport"
```

Comment:
A label is an extra widget.  I'd use the hint property of the component you are labeling instead.  Less clutter.